src/SignalR/clients/ts/FunctionalTests/package.json

```diff
@@ -5,16 +5,21 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
-    "@microsoft/signalr": "link:../signalr",
-    "@microsoft/signalr-protocol-msgpack": "link:../signalr-protocol-msgpack"
+    "@microsoft/signalr": "^5.0.0-dev",
```

Comment:
This looks sus, does it actually pick up the just built packages?

Reply:
I'm currently fixing this bit. It works because the packages define that version, but the correct thing is to do `*` and that will always pick the workspace.

Reply:
![image](https://github.com/dotnet/aspnetcore/assets/6995051/e45366de-a526-4d21-adb0-192b56a14fcf)
For reference the package from NPM
![image](https://github.com/dotnet/aspnetcore/assets/6995051/73c8f55e-b348-401b-ac75-3b30998f38d7)